app/components/SheetMusic/comments/Comments.jsx

```diff
@@ -0,0 +1,50 @@
+import fluxMixin from 'flummox/mixin';
+import React from 'react';
+
+import Comment from './Comment';
+import ReplyBox from './ReplyBox';
+
+function retrieveInitialData(flux, id) {
+  let sheetMusicActions = flux.getActions('sheetmusic');
+  return Promise.all([
+    sheetMusicActions.getComments(parseInt(id, 10), flux),
+  ]);
+}
+
+export default React.createClass({
+  propTypes: {
+    /**
+     * The id of the sheet music.
+     */
+    id: React.PropTypes.number.isRequired,
+  },
+
+  mixins: [
+    fluxMixin({
+      comments: store => ({
+        comments: store.state.comments,
+      }),
+    }),
+  ],
+
+  componentDidMount() {
+    retrieveInitialData(this.flux, this.props.id);
```

Comment:
Yeah, I'm more inclined towards making these comments components less "smart" (i.e not the ones making requests). Performing the get comments API call in the SheetMusic component could be one way of doing it. What if, to save the number of requests we're making, we modify the backend to return comments along with sheet music information? 

I feel like it's generally a good idea to have less "smart" components (i.e ones that make API calls) and more "dumb" components (i.e ones that just render stuff given some props) in our application. That's one of the reasons I really like Redux, since it takes care of all the smart component magic for you, and lets you only implement the dumb components. I think going forward we should structure our application in a way that allows easy migration to Redux.